File: PryBase/es.efor.PryBase.Angular/webapp/src/app/shared/api/models/user-dto.ts
/* tslint:disable */
import { UserDepartmentDto } from './user-department-dto';
import { UserLevelDto } from './user-level-dto';
export interface UserDto {
  department?: UserDepartmentDto;
  displayName?: null | string;
  id?: string;
  level?: UserLevelDto;
  name?: null | string;
}

File: PryBase/es.efor.PryBase.Infraestructure/DTO/UsersDTOs/UserDTO.cs
using System;

namespace es.efor.PryBase.Infraestructure.DTO.UserDTOs
{
    public class UserDTO
    {
        #region Constructor
        public UserDTO() { }
        #endregion

        #region Propiedades 
        public Guid Id { get; set; }
        public string Name { get; set; }
        public string DisplayName { get; set; }
        public UserDepartmentDTO Department { get; set; }
        public UserLevelDTO Level { get; set; }
        //public DateTime? DeleteDate { get; set; }
        //public string DeleteUser { get; set; }

        #endregion

    }
}
